libs/api/src/lib/graphql/blocks.ts

```diff
@@ -1103,27 +1102,29 @@ export const GraphQLEventBlock = new GraphQLObjectType<EventBlock, Context>({
     filter: {type: new GraphQLNonNull(GraphQLEventBlockFilter)},
     events: {
       type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLEvent))),
-      resolve: ({filter}, _, {prisma}) =>
```

Comment:
This was actually preparation of migration of events to v2. So then you don't rely on data form this, but only return ids with typename. Data is then retrieved via resolveReference on v2. 

However it can also coexist with types defined in v1 and v2 and both retrieving data on its own. Federation makes sure schema is unified.